crates/swc_plugin_runner/Cargo.toml

```diff
@@ -30,8 +30,7 @@ plugin_transform_host_native = [
   "wasmer-compiler-cranelift/default",
 ]
 plugin_transform_host_native_shared_runtime = [
-  "tokio",
-  "wasmer-wasix/webc_runner",
```

Comment:
this feature was removed as redundant?
https://github.com/wasmerio/wasmer/commit/0a8d91d7d6d79eaa2c026deeb572d0b440173e52